docs/ja/reference/function/before.md

```diff
@@ -5,7 +5,10 @@
 ## インターフェース
 
 ```typescript
-function before<F extends (...args: any[]) => any>(n: number, func: F): F;
+function before<F extends (...args: unkown[]) => any>(
```

Comment:
```suggestion
function before<F extends (...args: any[]) => any>(
```

Actually, since parameters are contravariant, we have to specify `never` or `any` in the parameters.

Reply:
Thanks to review!